src/Lexer.php

```diff
@@ -42,7 +53,7 @@ class Lexer
     private int $lineNumber;
     private bool $eos;
     /**
-     * @var array<string, string>
+     * @phpstan-var array<TKeywordsType, string>
```

Comment:
To avoid conflict with #350 where this prop is removed
```suggestion
     * @var array<string, string>
```